git-cliff-release/cliff.toml

```diff
@@ -38,7 +38,7 @@ body = """
             {% if commit.extra.commit_link %} ([{{ commit.id | truncate(length = 7, end = "") }}]({{ commit.extra.commit_link }})){% endif %}\
             {% if commit.remote.username and extra.is_release_notes %}\
             {{" "}}by @{{ commit.remote.username }}\
-            {% elif commit.remote.username %}\
+            {% elif not extra.no_github and commit.remote.username %}\
```

Comment:
Relies on the slightly unclear invariant `extra.no_github` => `!extra.is_release_notes` (actually `=> !extra.any-key-you-can-put-here`). Should we try to make this more readable, or is it okay for the scope of this project?

Reply:
Huh. Maybe we could copy `commit.remote.username` to `commit.extra.username` in `enhance_context.py` and use that here. Then the `extra.no_github` field would not even be needed in the release context.

Reply:
Right, so now `--no-github` just implicitly turns off all the extra properties. Neat, thanks!